Sources/PluginViewController/PluginViewController.swift

```diff
@@ -9,25 +9,38 @@ final class PluginViewController: UIViewController {
 
     var closeEventHandler: (() -> Void)?
 
-    private(set) lazy var tableView: UITableView = {
-        let tableView = UITableView(frame: .zero, style: .grouped)
-        tableView.delegate = configuration
-        tableView.dataSource = configuration
-        tableView.estimatedRowHeight = 100
-        tableView.separatorInset = .init(top: 0, left: 10, bottom: 0, right: 0)
-        return tableView
-    }()
 
     private let configuration: Configuration
 
     override var preferredStatusBarStyle: UIStatusBarStyle {
         return .default
     }
-    
+
     override var prefersStatusBarHidden: Bool {
         return false
     }
 
+    // MARK: - Subviews
+
+    private(set) lazy var searchBar: UISearchBar = {
```

Comment:
I guess using a `UISearchBar` directly here is fine, but why not use `UISearchController` instead? It'll look much cleaner.
We can switch to it later, though, it shouldn't be a blocker.